tests/lang/short_tags_unique_deprecation_notice.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+Unique PHP short tag deprecation with multiple short tags used
+--FILE--
+<?
+ echo "Used a short tag\n";
+?>
+<?php
+ echo "Use a normal tags\n";
+?>
+Non PHP content
+<?
+ echo "Another short tag\n";
+?>
+Finished
+--EXPECTF--

```

Comment:
Please change this test to first use a normal PHP tag, then short tag, and explicitly check the line number of the message. I'm not totally sure if it will be right.

Reply:
Going from Azure and Travis this looks to be working as expected. However, Appveyor is still pending because of queue backlog.